rpc/documentation/api.md

```diff
@@ -1371,7 +1372,7 @@ transaction using a wallet private keys.
   input that is missing an input script.
 
 - `repeated AdditionalScript additional_scripts`: Additional output scripts of
-  previous outputs spent by the transaction that the wallet may not be aware of.
+  previous outputs spent by the transactions that the wallet may not be aware of.
```

Comment:
"transaction" was correct here. This "transaction" refers to the transaction being signed, not to the transactions the wallet isn't aware of.